project/ProjectPlugin.scala

```diff
@@ -127,6 +128,21 @@ object ProjectPlugin extends AutoPlugin {
       )
     )
 
+    lazy val httpSettings: Seq[Def.Setting[_]] = Seq(
+      libraryDependencies ++= Seq(
+        %%("http4s-dsl", V.http4s),
+        %%("http4s-blaze-server", V.http4s),
+        %%("http4s-circe", V.http4s),
+        %%("circe-generic"),
+        "co.fs2" %% "fs2-reactive-streams" % V.reactiveStreams,
```

Comment:
If we are not offering the combinators that I've mentioned above, maybe we could get rid off this dependency.

Reply:
Even when we are not exposing the combinators, we still keep needing this dependency to perform some transformations.